index.html

```diff
@@ -69,7 +69,7 @@ <h1>Contact Us</h1>
     <footer>
       <p>Copyright Talha</p>
     </footer>
-    <script src="script.js"></script>
-    <script src="show.js"></script>
+    <script type="module" src="./index.js"></script>
+    <script src="./show.js"></script>
```

Comment:
- [ ] In the file `show.js` that you imported here, please, kindly refactor all your functions to use ES6 syntax.  That way, we will meet this requirement 💪